stl/inc/__msvc_chrono.hpp

```diff
@@ -666,18 +666,38 @@ namespace chrono {
         using time_point                = _CHRONO time_point<steady_clock>;
         static constexpr bool is_steady = true;
 
+#if defined(_M_ARM) || defined(_M_ARM64) // vvv ARM or ARM64 arch vvv
+#define _LIKELY_ARM _LIKELY
+#define _LIKELY_X86

```

Comment:
Metonymy is sometimes great, but it's confusing to use "ARM" to mean "ARM or ARM64" and especially to use "x86" to mean "x86 or x64". We should mention both here. (This is similar to `intrin.h`'s conventions.)